### PR TITLE
Mobilienet ssd analyzer tester

### PR DESCRIPTION
**Pull request by lidanqing-intel**
This is a PR new version for https://github.com/PaddlePaddle/Paddle/pull/17738
Because https://github.com/PaddlePaddle/Paddle/pull/17738 is old and a lot of new commits are added. This one is based on newest develop.
In local build directory `ctest -R test_analyzer_int8_mobilenet_ssd` passed. Waiting for CI.

### CODE REVIEW

**Comment by lidanqing-vv** [comment]
Hi,  @luotao1  I opened a new PR and closed PR17738.

**Comment by lidanqing-vv** [comment]
@wojtuss @sfraczek @Sand3r- @zhaify Hi, could you please help to review. It is said this will be used as a testing demo in Baidu Create Meeting. I wish I could get some reviews early and make changes on time. Thank you very very much.

**Comment by lidanqing-vv** [comment]
I will add local-converting function in `full_pascalvoc_test_preprocess.py` and  full_ILSRVOC_test_preprocess.py` (in next PR) because validation team asked me why we always have to download and convert all each time and could not converted as many as we need. I will add this option.

File: paddle/fluid/inference/tests/api/full_pascalvoc_test_preprocess.py [hunk 1]
```diff
@@ -0,0 +1,187 @@
+# Copyright (c) 2016 PaddlePaddle Authors. All Rights Reserved
```
Comment (wojtuss) [inline paddle/fluid/inference/tests/api/full_pascalvoc_test_preprocess.py]: 2016 -> 2019

Comment (lidanqing-vv) [reply paddle/fluid/inference/tests/api/full_pascalvoc_test_preprocess.py]: Thanks. Now it's changed.

**Comment by luotao1** [comment]
```
44/160 Test #164: test_analyzer_int8_mobilenet_ssd .............   Passed  117.10 sec
```
http://ci.paddlepaddle.org/viewLog.html?buildId=114512&buildTypeId=Paddle_PrCi&tab=buildLog&_focus=12462#_state=83

File: paddle/fluid/inference/tests/api/CMakeLists.txt [hunk 2]
```diff
@@ -159,55 +159,70 @@ if(WITH_MKLDNN)
   if (NOT EXISTS ${INT8_DATA_DIR})
     inference_download_and_uncompress(${INT8_DATA_DIR} "${INFERENCE_URL}/int8" "imagenet_val_100_tail.tar.gz")
   endif()
+  
+  if (NOT EXISTS ${INT8_DATA_DIR}/pascalvoc_data.bin)
+    inference_download_and_uncompress(${INT8_DATA_DIR} "${INFERENCE_URL}/int8" "pascalvoc_val_200_head.tar.gz")
```
Comment (luotao1) [inline paddle/fluid/inference/tests/api/CMakeLists.txt:164]: I see `imagenet_val_100_tail` has 100 images, but `pascalvoc_val_200_head` has 200 images, could you change to `pascalvoc_val_100_head`?

Comment (lidanqing-vv) [reply paddle/fluid/inference/tests/api/CMakeLists.txt:164]: ok ~

**Pull request merged**